google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -26,31 +27,46 @@
 
 @dataclasses.dataclass
 class Entity:
-    r"""Represents a wrapped documentai.Document.Entity.
+    """Represents a wrapped `documentai.Document.Entity`.
 
     Attributes:
         documentai_entity (google.cloud.documentai.Document.Entity):
-            Required. The original google.cloud.documentai.Document.Entity object.
+            Required. The original `google.cloud.documentai.Document.Entity` object.
+        page_offset (int):
+            Optional. The page offset (for sharded Documents) of the `google.cloud.documentai.Document` containing
+            the `google.cloud.documentai.Document.Entity`. `page_refs.page` is relative to the shard, not the entire Document.
```

Comment:
`page_refs` seems to be an attribute of the underlying documentai message and is used only in the implementation, and this part of this docstring would not help the user understand what `page_offset` means.

Reply:
Rewrote this doc string. Is this more clear?